### PR TITLE
adding simple play for role creation

### PR DESCRIPTION
**Pull request by e0d**

### CODE REVIEW

File: playbooks/ansible_role.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- name: Configure instance(s)
```
Comment (feanil) [inline playbooks/ansible_role.yml:1]: Can you add a comment on how to pass in the variable for the role name and a sample ansible command.

Comment (jarv) [inline playbooks/ansible_role.yml:1]: it's handy to run plays like this without an inventory script. The way I have been running plays run on localhost is setting:
`hosts: all` in the play and then:

`ansible-playbook -c local -i "localhost," ansible_role.yml`

**Comment by jarv** [comment]
completely forgot that there was already an open pull for this, I think this can be closed since I just merged in a play that does this.

**Comment by jarv** [comment]
closing

**Pull request closed without merging**